pkg/controller/jobs/rayjob/rayjob_controller.go

```diff
@@ -22,6 +22,7 @@ import (
 	"strings"
 
 	rayv1 "github.com/ray-project/kuberay/ray-operator/apis/ray/v1"
+	raycommon "github.com/ray-project/kuberay/ray-operator/controllers/ray/common"
```

Comment:
The vendor changes are likely due to this import. I can remove it and just copy the GetDefaultSubmitterTemplate function instead if the vendor changes are problematic

Reply:
This change actually makes me not comfortable including this for 0.10 in the last minute. It also makes me hesitant to cherry-pick to 0.9.2. I would propose to find a fix with this bump. We can then bump in a follow up meant for 0.11